src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoCommand.cs

```diff
@@ -471,6 +472,54 @@ private static List<MediaInfo> QueryMedia(SelectArguments arguments)
             }
         }
 
+        /// <summary>
+        /// Retrieves all matched ebook path with given <paramref name="keyword"/>.
```

Comment:
ebook path -> ebook paths, same in \<returns\>

Reply:
Done.